spec/models/jsonld_record_spec.rb

```diff
@@ -24,12 +24,14 @@
       'provenance_display'         => ['Provenance: Johann Anton André, of Offenbach, 1799; Philharmonische Gesellschaft, Laibach (Ljubljana); Robert Ammann.'],
       'source_acquisition_display'         => ['Obtained, Nov. 16, 1961, at Stargardt Sale of the collection of Dr. Robert Ammann.'],
       'references_display'         => ["Stillwell B460.", "Goff B-526."],
-      'electronic_access_1display' => ['{"http://arks.princeton.edu/ark:/88435/dr26z114k":["arks.princeton.edu"],"http://digital.lib.cuhk.edu.hk/crbp/servlet/list":["First page of main text"]}'],
+      'electronic_access_1display' => ["{\"http://arks.princeton.edu/ark:/88435/47429918s\":[\"arks.princeton.edu\",\"Finding aid\"],\"http://arks.princeton.edu/ark:/88435/7p88ch283\":[\"arks.princeton.edu\"]}"],
```

Comment:
Maybe a comment here or in the spec since this doesn't have its own spec documenting the functionality